docs/deployment-notes/puppeteer/configure-page-layout-assignments.mjs

```
@@ -0,0 +1,142 @@
+#!/usr/bin/env node
+/* eslint-env node */
+
+'use strict';
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)